data/locale/sv.ts

```diff
@@ -159,7 +159,7 @@ If you&apos;re interested in translating LMMS in another language or want to imp
     </message>
     <message>
         <source>Continue sample playback across notes</source>
-        <translation>Forsätt spela ljudfil över noter</translation>
+        <translation>Forsätt sampla ljudfilen över noter</translation>
```

Comment:
I think this is incorrect, the meaning isn't the same. (Ljudfil -> Ljudfilen I agree on though, that was my mistake)

"Sample playback" = "Playing the sample" = "Spela ljudfilen" != "Sampla" = "Sampling" = "Using samples"

Of course I could be wrong, my Swedish is worse than my English.